### PR TITLE
docs: update leaderboards

### PR DESCRIPTION
**Pull request by Asteric0**
is dis good

### CODE REVIEW

File: chatreactions/leaderboards.md [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+type: `sailing`
+
+**winners**
+
+🥇Ayestk in `1.89s`
+
+🥈Scarlett in `2.73s`
+
+🥉M.axz in `3.42s`
+
```
Comment (mxz7) [inline chatreactions/leaderboards.md:18]: maybe change this to a picture, put pictures in the assets folder.

**Review by mxz7 — CHANGES_REQUESTED** [review]
make everything lowercase please (:

File: chatreactions/leaderboards.md (hunk 1 above)
Comment (Asteric0) [reply chatreactions/leaderboards.md:18]: why lower case?

**Comment by mxz7** [comment]
Are you going to rectify the issues?

**Pull request closed without merging**